tests/test_markupsafe.py

```diff
@@ -80,12 +80,13 @@ def test_escaping(escape: TPEscape) -> None:
     assert escape("\"<>&'") == "&#34;&lt;&gt;&amp;&#39;"
     assert (
         Markup(
-            "<!-- outer comment -->"
+            " <!-- outer comment with leading and trailing whitespace -->\n"
             "<em>Foo &amp; Bar"
-            "<!-- inner comment about <em> -->"
+            " <!-- inner comment about <em> --> "
```

Comment:
Maybe not replace the original `outer` and inner comment lines, but add the new lines (those which are with extra whitespace) in addition to them? It should increase the coverage a bit.

Reply:
By "coverage" I meant exercising both the case when a comment is surrounded by spaces and when it's not. Though I'm not really sure that this is really needed. :) Maybe maintainers will have an opinion here. As for me,  it's fine in either case.
Thanks.